broker/cloud_run/lsst/variability/main.py

```diff
@@ -0,0 +1,220 @@
+    except pittgoogle.exceptions.BadRequest as exc:
+        return str(exc), HTTP_400
+
+    TOPIC.publish(
+        pittgoogle.Alert.from_dict(
+            {**alert_lite.dict, "variability": _calculate_stetsonJ_statistics(alert_lite)},
+            attributes={**alert_lite.attributes, "value_added": "variability"},
+            schema_name="default",
+        )
+    )
+
+    return "", HTTP_204
+
+
+def _calculate_stetsonJ_statistics(alert_lite: pittgoogle.Alert) -> Dict:
+    """Adapted from:
+    https://github.com/lsst/meas_base/blob/e5cf12406b54a6312b9d6fa23fbd132cd7999387/python/lsst/meas/base/diaCalculationPlugins.py#L904
+
+    Compute the StetsonJ statistics on the DIA point source fluxes for each band.
+    """
+    alert_lite_dict = alert_lite.dict["alert_lite"]
+    alert_df = _create_dataframe(alert_lite_dict)
+    bands = alert_df["band"].unique()
+    outgoing_dict = {
+        "diaObjectId": alert_lite_dict["diaObject"]["diaObjectId"],
+        "diaSourceId": alert_lite_dict["diaSource"]["diaSourceId"],
+    }
+
+    # filter diaSource(s) in alert_df based on the filter(s) used
+    for band in bands:
+        filter_diaSources = alert_df[alert_df["band"] == band]
+        tmp_df = filter_diaSources[
+            ~np.logical_or(
+                np.isnan(filter_diaSources["psfFlux"]),
+                np.isnan(filter_diaSources["psfFluxErr"]),
+            )
+        ]
+
+        if len(tmp_df) < 2:
+            outgoing_dict[f"n_detections_{band}_band"] = len(tmp_df)
+            outgoing_dict[f"{band}_psfFluxStetsonJ"] = np.nan
+            continue
+
+        fluxes = tmp_df["psfFlux"].to_numpy()
+        errors = tmp_df["psfFluxErr"].to_numpy()
+        outgoing_dict[f"n_detections_{band}_band"] = len(tmp_df)
+        outgoing_dict[f"{band}_psfFluxStetsonJ"] = _stetson_J(fluxes, errors)
+
+    return outgoing_dict
+
+
+def _create_dataframe(alert_dict: pittgoogle.Alert) -> "pd.DataFrame":
+    """Return a pandas DataFrame containing the source detections."""
+
+    # sources and previous sources are expected to have the same fields
+    sources_df = pd.DataFrame(
+        [alert_dict.get("diaSource")] + (alert_dict.get("prvDiaSources") or [])
+    )
+    # sources and forced sources may have different fields
+    forced_df = pd.DataFrame(alert_dict.get("prvDiaForcedSources") or [])
+
+    # use nullable integer data type to avoid converting ints to floats
+    # for columns in one dataframe but not the other
+    sources_ints = [c for c, v in sources_df.dtypes.items() if v == int]
+    sources_df = sources_df.astype(
+        {c: "Int64" for c in set(sources_ints) - set(forced_df.columns)}
+    )
+    forced_ints = [c for c, v in forced_df.dtypes.items() if v == int]
+    forced_df = forced_df.astype({c: "Int64" for c in set(forced_ints) - set(sources_df.columns)})
+
+    _dataframe = pd.concat([sources_df, forced_df], ignore_index=True)
+    return _dataframe
```

Comment:
Is this redundant with `Alert.dataframe`? (Maybe we already talked about it and I'm not remembering.)

Reply:
@troyraen What do you mean? Providing this description below to add context:

I adapted this function from the pittgoogle-client since I'm unable to directly use `alert_df = alert_life.dataframe` (as you brought up on Slack, `.dataframe` relies on the schema map, i.e., `self.get(...)`).

So this is a quick patch while the other issues we talked about today get addressed.

Reply:
> So this is a quick patch while the other issues we talked about today get addressed.

Great, ya that was my question. I realize now that you said exactly this this morning, but I was looking at this code thinking it pre-existed that insight/decision so I didn't quite connect the two.